src/OOVPADatabase/D3D8LTCG/5849.inl

```diff
@@ -742,6 +742,23 @@ OOVPA_NO_XREF(D3DDevice_CreatePixelShader, 1024, 12)
     { 0x2B, 0xC2 },
 } OOVPA_END;
 
+// ******************************************************************
+// * D3DDevice_SetStreamSource
+// ******************************************************************
+OOVPA_XREF(D3DDevice_SetStreamSource_0__LTCG_eax_StreamNumber_edi_pStreamData_ebx_Stride, 2058, 1+12,
+
+    XRefNoSaveIndex,
+    XRefOne)
+{
+
+        XREF_ENTRY( 0x19, XREF_G_STREAM ), // Derived
+
+        // test edi, edi; mov ecx, [...]
+        OV_MATCH(0x00, 0x85, 0xFF, 0x8B, 0x0D),
```

Comment:
Insert new line below this line. Why? Above OV_MATCH only has 4 bytes whilst next OV_MATCH is at offset 8. Meaning you skip over 4 bytes which require extra new line spacing.